python/cudf/cudf/core/multiindex.py

```diff
@@ -615,9 +616,10 @@ def __eq__(self, other):
     def is_contiguous(self):
         return True
 
+    @copy_docstring(Frame.size)
```

Comment:
Do we need to do this? Doesn't Sphinx use the docstring of `Frame` here automatically? Python/IPython `help` does that even without `copy_docstring`.

Reply:
Yep, this was not needed - Removed it. Sphinx picks the docs the same way help does.